catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/model/Registry.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+    'wreqr',
+    'q',
+    'js/model/Service.js',
+    'js/model/RemoteStatus.js',
+    'backbone',
+    'underscore',
+    'poller',
+    'js/model/Status.js',
+    'backboneassociation'
+],
+function (wreqr, Q, Service, RemoteStatus, Backbone, _, poller, Status) {
+    var Registry = {};
+
+
+    Registry.ConfigurationList = Backbone.Collection.extend({
+        model: Service.Configuration
+    });
+
+    Registry.Model = Backbone.AssociatedModel.extend({
+        defaults: {
+                registryConfiguration : []
+        },
+        configUrl: "/admin/jolokia/exec/org.codice.ddf.ui.admin.api.ConfigurationAdmin:service=ui",
+        idAttribute: 'name',
+        remoteId: 'remoteName',
+        pullAttribute: 'pullAllowed',
+        pushAttribute: 'pushAllowed',
+        pidAttribute: 'pid',
+        initialize: function() {
+            this.set('registryConfiguration', new Registry.ConfigurationList());
+        },
+        addRegistryConfiguration: function(registry) {
+            this.get("registryConfiguration").add(registry);
+
+            var pid = registry.id;
+            var statusModel = new Status.Model(pid);
+            var remoteStatusModel = new RemoteStatus.Model(pid);
+            statusModel.on('sync', function() {
```

Comment:
So after talking with @andrewkfiedler we moved the poller logic out of the model and into the view. This eliminates the need for wreqr in the model class. The view now stops the pollers and disconnects the events when onClose() is called on the view.